### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by albert-ying** [piyueh/totero#2]
The TUI looks great and I really like the vim binding. However, It is not very useful at the current stage as some essential functions are missing (e.g., searching). I'm wondering if you are going to keep developing this?

**Comment by piyueh** [comment]
@albert-ying
Yes, kind of abandoned. I don't have time to add new features. Nevertheless, I'll still fix bugs if anyone opens new issues.

**Comment by albert-ying** [comment]
It's a pity as it's a such good idea. Maybe you can update the status to README and see if anyone would like to help (I want to help if I could, but unfortunately I'm not proficient in python).

**Comment by piyueh** [comment]
Thanks for the suggestion! I updated the README.
